### ISSUE TITLE
Feature Request - the ability to strip parameters from urls so duplicate files are not created

### ISSUE THREAD

**Issue by pdwalker** [xroche/httrack#225]
as described here: 
https://forum.httrack.com/readmsg/27508/index.html

a simple solution using a wrapper
https://forum.httrack.com/readmsg/36962/27508/index.html

wrapper code here:
https://gist.github.com/krzys-h/4717608089c54f733083fd390e5c0f2b

Ideally, a more generic solution would be added so one more more url parameters could be dealt with.

**Comment by mitchcapper** [comment]
Agreed on this @pdwalker , there are already url filters like "-*logout*"  was thinking potentially "-#?param_name"  which is introducing the new "#" extension (could be something else invalid in a true url like "|" ) and then the ? to indicate a query parameter. This would allow for other special operators than just ? down the line.   The other option would be some standard arg format like `--strip_params "sid,session_cookie,cache_bust"`  curious on thoughts

**Comment by xroche** [comment]
Duplicate of #112 (query-aware filtering), which tracks stripping or normalizing URL query parameters to avoid duplicate files. Closing in favour of #112, where the `--strip-params`-style proposal here is captured.
